tests/visual/progressbar.html

```diff
@@ -161,22 +362,52 @@
                 </ul>
             </div>
 
-            <span></span>
-            <span>LTR</span>
-            <span>RTL</span>
-            <span>Indeterminate</span>
```

Comment:
Why did you remove the Indeterminate progressbar?

Reply:
re-added